shim/src/epoll/epoll_ctl.c

```diff
@@ -16,6 +16,9 @@
 
 static int __do_demi_epoll_ctl_add(int epfd, int fd, struct epoll_event *event)
 {
+    int i;
```

Comment:
What is this variable used for?

Reply:
It was a habit of declaring variables in the beginning of the function. Some recent projects I was working on followed this code style. There are some other comments related to this. I will keep the declarations in the inner scope when appropriate (such as inside the for loop in this instance), to be consistent with the rest of the code in the demikernel.